src/pages/add/search.vue

```diff
@@ -18,7 +18,7 @@
             <TextFieldSingleLine
               v-model.trim="searchWord"
               @enterTextField="search(0)"
-              placeholder="科目名または科目番号"
+              placeholder="授業名や科目番号  (例 :「情報  倫理」,「GA」 など)  "
```

Comment:
```suggestion
              placeholder="授業名や科目番号  (例 :「情報 倫理」,「GA」など)"
```
わたしがデザインデータ上で余分なスペースを入れてしまっていたようなので消させてもらいました…さーせん…(一応、ここ以外の複数スペースは意図的です)